nttt/utilities.py

```diff
@@ -53,4 +54,29 @@ def find_files(src, file_names=[], extensions=[]):
             if valid_file:
                 files_found.append(fpath)
 
-    return files_found
\ No newline at end of file
+    return files_found
+
+
+#PUB-9 utilities
+def special_trim_pub_9(mystring):
+    mystring = mystring.replace("** ", "**")
+    mystring = mystring.replace(" **", "**")
+    mystring = mystring.replace("* ", "*")
+    mystring = mystring.replace(" *", "*")
+    mystring = mystring.replace("_ ", "_")
+    mystring = mystring.replace(" _", "_")
+    mystring = mystring.replace("` ", "`")
+    mystring = mystring.replace(" `", "`")
+    return mystring
+
+def find_candidate_strings_for_pub_9(mystring):
+    candidates1 = re.findall(r"_ [a-z]* _", mystring)
+    candidates2 = re.findall(r"` [a-z]* `", mystring)
+    candidates3 = re.findall(r"[*]+ [a-z]* [*]+", mystring)
+    return candidates1 + candidates2 + candidates3
+
+def trim_pub_9(mystring):
```

Comment:
Please use function names that describe what the function does. For example, this function could be called trim_special_markdown_tags. Another developer (and probably also you in a couple of days/weeks/months) will not know by heart what pub_9 is, and will have to search in Jira or somewhere else to find out what this function does.